lib/ReactJsonSchema.js

```diff
@@ -18,19 +18,25 @@ export default class ReactJsonSchema {
   parseSubSchemas(subSchemas = []) {
     const Components = [];
     let index = 0;
-    for (const subSchema of subSchemas) {
+    Object.keys(subSchemas).forEach((key) => {
+      const subSchema = subSchemas[key];
       subSchema.key = typeof subSchema.key !== 'undefined' ? subSchema.key : index;
       Components.push(this.parseSchema(subSchema));
       index++;
-    }
+    });
     return Components;
   }
 
   createComponent(schema) {
-    const { component, children, text, ...rest } = schema;
```

Comment:
What benefit does removing this provide?

Reply:
Hey @wtgtybhertgeghgtwtg, thanks for looking. 

I originally did this I believe because eslint was complaining about the unused-vars. I thought that it was innocent enough to just delete the props instead. It looks like deleting may actually have a slight performance impact so I'm reverting back and just slapping an eslint ignore comment for the unused vars. 

Thanks again.